src/course-search/TypeaheadFacetDropdown.jsx

```diff
@@ -43,6 +44,7 @@ const TypeaheadFacetDropdown = ({
       title={title}
       isBold={isBold}
       type="typeahead"
+      {...props}
```

Comment:
For my own learning: it feels like spread operator with {} around it should cancel each other out. Are you doing this as a cheap way of making an immutable copy of this data?

Reply:
No, it's inside a JSX tag, so the spread operator here will take whatever is in props and add them as props to this component. So, if what's in props is `{ color: 'blue', variant: 'yellow'}` this would be the same as writing
```
color='blue'
variant='yellow'
``` 
on these lines.

I suspect the syntactic sugar of JSX is turning the everything above this into an object somewhere under the hood, anyway, and this is just extending that object. 
It's... not the choice I would have made had I written React. But here we are.